tower_cli/resources/notification_template.py

```diff
@@ -163,6 +164,12 @@ def _separate(self, kwargs):
                     except ValueError:
                         raise exc.TowerCLIError('Provided json file format '
                                                 'invalid. Please recheck.')
+                    except TypeError:
+                        # If result[field] is already in json I am getting an exception of:
+                        #    'expected string or buffer'
+                        # Since it was not caught it caused an error programatically creating a notification template.
+                        # So we are just going to catch a generic exception and pass on it.
+                        pass
```

Comment:
I think it would be cleaner to catch this particular case with a check before converting the data like

```python
if isinstance(result[field], dict):
    continue
```

And it would avoid unexpected silent failures. If you have to keep this, it would be good to write a debug statement for it.

Reply:
Instead of doing isistance(..., dict) I went with not isinstance(..., str) since json.loads expects a str.